README.md

```diff
@@ -71,6 +100,26 @@ make install
 make run
 ```
 
+Please remember the `ARGOCD_AUTH_TOKEN` and `ARGOCD_SERVER_ADDR` variables need to be present in order 
+to run against a Kubernetes cluster with Argo CD. If the cluster was configured using the `hack/setup-dev.sh` script,
+these variables are part of the `.env.local` file.
+
+
+### Deploying to a Kubernetes cluster
+
+To deploy the controller to a Kubernetes cluster, run:
+```
+make install
+make docker-build
+make deploy
+```
+
+In order to so, a `.env.cluster` file needs to exist in the `config/manager` folder, containing
```

Comment:
I am not sure about adding a file into the kustomize directory. Is that the only way to make kustomize work?

Reply:
Not very fond of it either, to be honest. But from https://kubectl.docs.kubernetes.io/faq/kustomize/: `v2.0 added a security check that prevents kustomizations from reading files outside their own directory root.` 

This meant I can't find any file that's not inside that directory - I'm open to alternatives.

Reply:
Ah, so if I had not stopped reading at the first line, it says this: 

```Resources (including configmap and secret generators) can still be shared via the recommended best practice of placing them in a directory with their own kustomization file, and referring to this directory as a base from any kustomization that wants to use it. This encourages modularity and relocatability.```

Let me investigate that.